robottelo/utils/url.py

```diff
@@ -1,9 +1,55 @@
 from urllib.parse import urlparse
 
+from robottelo.logging import logger
+
 
 def is_url(url):
     try:
         result = urlparse(url)
         return all([result.scheme, result.netloc])
     except (ValueError, AttributeError):
         return False
+
+
+def is_ipv4_url(text):
+    # Did not find the better way to filter only URLs so skipping it simple
+    # and open for reviewers suggestions
+    if isinstance(text, str) and 'ipv4' in text and 'redhat.com' in text:
```

Comment:
```suggestion
    if isinstance(text, str) and '.ipv4.' in text:
```

Reply:
@JacobCallahan There is no dots around ipv4. Mostly they are `-ipv4`. That is hostname appended with `-ipv4`

Reply:
@ogajduse You mean, you are in favor of keeping `redhat.com`.

Reply:
I have moved the condition to check `if '-ipv4' in text`.